tests/phpunit/tests/test-wpml-elementor-translatable-nodes.php

```diff
@@ -385,4 +385,804 @@ public function it_updates() {
 
 		$this->assertEquals( $translation, $element['settings']['editor'] );
 	}
+
+	/**
+	 * @test
+	 * @group wpmlcore-5803
+	 */
+	public function it_gets_single_repeater_field() {
+		$subject = new WPML_Elementor_Translatable_Nodes();
+		$field_id = 1;
+
+		$nodes = array(
+			'my-custom-module'      => array(
+				'conditions'        => array( 'widgetType' => 'my-custom-module' ),
+				'fields'            => array(),
+				'integration-class' => 'WPML_PB_My_Custom_Module_With_A_Single_Repeater_Field',
+			),
+		);
+
+		WP_Mock::onFilter( 'wpml_elementor_widgets_to_translate' )
+		       ->with( $this->get_translatable_nodes() )
```

Comment:
@adrianosferreira, I am not sure if we should really pass all the nodes again (it's probably already covered in your other tests). Maybe you can use the code below (not tested...) as a suggestion.

```
->with( \WP_Mock\Functions::type( 'array' ) )->reply( $nodes )
```

Reply:
@strategio I didn't want to have these nodes, but unfortunately, I didn't find any alternative. Your idea doesn't work, the filter doesn't reply with my nodes.